src/app.module.ts

```diff
@@ -1,12 +1,16 @@
-import {Module, ValidationPipe} from '@nestjs/common';
-import {ConfigModule} from '@nestjs/config';
-import {APP_PIPE} from '@nestjs/core';
+import { Module, ValidationPipe } from '@nestjs/common';
+import { ConfigModule } from '@nestjs/config';
+import { APP_INTERCEPTOR, APP_PIPE } from '@nestjs/core';
 import * as Joi from 'joi';
-import {LoggerModule} from 'nestjs-pino';
+import { LoggerModule } from 'nestjs-pino';
 import appConfig from './config/app.config';
-import {ReviewsModule} from './reviews/reviews.module';
-import {KafkaModule} from './kafka/kafka.module';
-import {PrismaModule} from './prisma/prisma.module';
+import { ReviewsModule } from './reviews/reviews.module';
+import { KafkaModule } from './kafka/kafka.module';
+import { PrismaModule } from './prisma/prisma.module';
```

Comment:
Let's follow the ladder import structure here.

Also, it's a good idea to group the imports based on core libraries, third-party libraries, and modules.